dts/bindings/mhu/arm,mhu.yaml

```diff
@@ -0,0 +1,27 @@
+---
+title: ARM MHU
+version: 0.1
+
+description: >
+    This binding gives a base representation of the ARM MHU
+
+properties:
+    compatible:
+      type: string
+      category: required
+      description: compatible strings
+      constraint: "arm,mhu"
+      generation: define
+
```

Comment:
add a label property here

Reply:
Added the MHU label, and use it in the code, commit 51ac287cffd90214aef115b649a71fda6873f669.
Thanks